### DISCUSSION TITLE
Welcome to RobinHood Discussions!

### DISCUSSION THREAD

**Discussion by JestonBlu** [JestonBlu/RobinHood#136] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by amacbale** [comment]
Hi there! First of all, thank you so much for all the work you've done on the RobinHood package!

I was wondering about crypto trading with RobinHood. I've written the skeleton of my program, but now when I try to place a real order, I find that I'm unable. Trying to place an order, I get HTTP 400 errors from the call to api_orders_crypto(). Is this known? It seems no one else in the entire world is using R to trade crypto through Robinhood so I'm not sure where else to turn. I've investigated the function definitions and tried to track down where this issue is coming from. My suspicion is that it has something to do with the Nummus API no longer functioning with Robinhood. Starting to doubt that crypto trading with Robinhood via R is possible. 

Are these known issues with crypto trading?

Thank you!

**Comment by ChaimtR** [comment]
Hi there! 
Im wondering about getting trading history. I used the 'export_completed_stock' function to get my trades but it seems to place my partially filled orders as fully completed orders. Any ideas on getting the partial orders correct?
